tests/framework/widgets/ActiveFieldTest.php

```diff
@@ -691,6 +692,16 @@ public function testInputOptionsTransferToWidget()
             'options' => ['placeholder' => 'pholder_both_direct']
         ]);
         $this->assertStringContainsString('placeholder="pholder_both_direct"', (string) $widget);
+
+        try {
+            $widget = $this->activeField->widget(TestMaskedInput::className(), [
```

Comment:
Why is there no assertion to test when no exception is thrown? Why also did you add try...catch?

Reply:
Perhaps you can tell me what exactly needs to be tested in this case? After all, previously the initialization of the widget was not covered by the test.

Also, I have never written tests before)

Reply:
You are basically testing the case where  you configure only regex things should go fine. So it is similar to previous tests (see example before) only different configurations are being tested

 ```
        $widget = $this->activeField->widget(TestMaskedInput::className(), [
            'mask' => '999-999-9999',
            'options' => ['placeholder' => 'pholder_direct'],
        ]);
        $this->assertStringContainsString('placeholder="pholder_direct"', (string) $widget);
```

Reply:
You do not need to try/catch as exception will cause the test to fail anyway